scaffolding-chef-infra/lib/linux/run.bash

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+export SSL_CERT_FILE="{{ pkgPathFor "${scaffold_cacerts}" }}/ssl/cert.pem"
+export SSL_CERT_DIR="{{ pkgPathFor "${scaffold_cacerts}" }}/ssl/certs"
```

Comment:
This variable is templated at build time. We are using $scaffold_cacerts here as a template. Please see this code: https://github.com/chef/effortless/blob/master/scaffolding-chef-infra/lib/linux/scaffolding.sh#L52

The default is always set in the build. See here: https://github.com/chef/effortless/blob/master/scaffolding-chef-infra/lib/linux/scaffolding.sh#L17

Reply:
Nope. I'm sorry but that bash code you sent me is only checking if the variable exists, add it as a package dep `pkg_deps`, else you add "the default `core/cacerts`". 
```bash
  if [ -n "${scaffold_cacerts}" ]; then
    pkg_deps+=("${scaffold_cacerts}")
  else
    pkg_deps+=("core/cacerts")
  fi
```
**But you never set the variable to any default.**

Reply:
Ah I see, `export SSL_CERT_FILE="{{ pkgPathFor "${CFG_CACERTS:-core/cacerts}" }}/ssl/cert.pem"`

This is how it was setting the variable default previously. I understand how this was confusing because of the double-build that was using the heredoc as a method of compiling these variables together.

We'll need to set a default in the sed command so it can template out successfully.